### PR TITLE
ENH: Pre-cache DKT31 template in Docker image

### PR DESCRIPTION
**Pull request by ZhifangYe**
This PR solved #1145 by cache DKT31 atlas in container. It also modified `cifti.py` line133-136 to respect the naming convention.

I think the check will fail until poldracklab/niworkflows#245 merged.

### CODE REVIEW

File: docker/scripts/get_templates.sh [hunk 1]
```diff
@@ -8,6 +8,8 @@ OASIS_TEMPLATE="https://files.osf.io/v1/resources/fvuh8/providers/osfstorage/584
 OASIS_SHA256="d87300e91346c16f55baf6f54f5f990bc020b61e8d5df9bcc3abb0cc4b943113"
 NKI_TEMPLATE="https://files.osf.io/v1/resources/fvuh8/providers/osfstorage/59cd90f46c613b02b3d79782"
 NKI_SHA256="4bba067f6675d15be96b205cb227e18a540673fd7e4577e13feedcef3a6f0ec5"
+OASIS_DTK31_TEMPLATE="https://files.osf.io/v1/resources/fvuh8/providers/osfstorage/5b16f17aeca4a80012bd7542"
```
Comment (effigies) [inline docker/scripts/get_templates.sh]: You use `OASIS_DTK31_TEMPLATE` here, but `$OASIS_DTK31_ATLAS` below. Either one is fine with me.

Comment (ZhifangYe) [reply docker/scripts/get_templates.sh]: Thanks.
Fixed now.

**Comment by effigies** [comment]
I added pins so that your modifications to niworkflows get included in the build. If this passes, then we will merge poldracklab/niworkflows#245, and we'll be able to release and update this PR to depend on it.

**Comment by effigies** [comment]
Travis apparently maintains its own pip cache, so it's failing to build. As long as Circle builds, I'm happy to merge, and we can get a final test on master. @chrisfilo Sound okay, or should we wait on the whole build?

**Pull request merged**